### PR TITLE
Remove Conductor, its workers and dependencies

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Comment by tranchitella** [comment]
Pipeline started manually:
https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/124763523

It includes:
https://github.com/mendersoftware/tenantadm/pull/131
https://github.com/mendersoftware/deviceauth/pull/314
https://github.com/mendersoftware/mender-qa/pull/375

**Comment by lluiscampos** [comment]
I had a look and so far looks okey, but I assume there are integration test changes missing in this PR. Is it still WIP or do you want me to dig into which changes I find missing?

**Comment by tranchitella** [comment]
@lluiscampos this is going to be a complex task, not ready for review. We are fixing here and there to have the tests running green.

**Comment by tranchitella** [comment]
@lluiscampos ready for review, back-end tests are green, full integration tests are tricky because of meta-mender but should be okay, I don't expect big changes here.

**Comment by lluiscampos** [comment]
The backend-tests require some reformatting, see https://gitlab.com/Northern.tech/Mender/integration/-/jobs/467135938

File: component-maps.yml [hunk 1]
```diff
@@ -82,24 +82,6 @@ git:
...
-    release_component: true
-
-  mender-conductor-enterprise:
-    docker_image:
-    - mender-conductor-enterprise
-    - org-welcome-email-preparer
-    docker_container:
-    - mender-conductor
-    - mender-org-welcome-email-preparer
-    release_component: true
```
Comment (kacf) [inline component-maps.yml]: In `component-maps.yml`, these should not be removed, but should be set to `release_component: false`. This has to do with being able to query older versions integration. It must still be able to answer this query, for example:

```
release_tool.py --version-of mender-conductor --in-integration-version 2.0.0
```

A comment might be appropriate though (I think YML supports that).

Same for the rest of the changes in `component-maps.yml`.

File: testutils/infra/container_manager/docker_compose_manager.py [hunk 2]
```diff
@@ -73,8 +73,8 @@ class DockerComposeNamespace(DockerNamespace):
         COMPOSE_FILES_PATH + "/extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml",
     ]
 
-    NUM_SERVICES_OPENSOURCE = 14
-    NUM_SERVICES_ENTERPRISE = 18
+    NUM_SERVICES_OPENSOURCE = 11
+    NUM_SERVICES_ENTERPRISE = 13
```
Comment (kacf) [inline testutils/infra/container_manager/docker_compose_manager.py]: Maybe you know already, but I didn't see it, so just FYI: This needs to be changed in [mender-docs](https://github.com/mendersoftware/mender-docs/blame/master/07.Administration/02.Production-installation/docs.md#L654) as well. There are several of those, this is a good expression to find them:

```bash
git grep -e "grep -c -i 'up'"
```

File: extra/release_tool.py [hunk 3]
```diff
@@ -193,8 +191,8 @@ def is_release_component(self):
 # categorize backend services wrt open/enterprise versions
 # important for test suite selection
 BACKEND_SERVICES_OPEN = {"deviceauth", "inventory"}
-BACKEND_SERVICES_ENT = {"deployments-enterprise", "mender-conductor-enterprise", "tenantadm", "useradm-enterprise"}
-BACKEND_SERVICES_OPEN_ENT = {"deployments", "mender-conductor", "useradm"}
+BACKEND_SERVICES_ENT = {"deployments-enterprise", "tenantadm", "useradm-enterprise"}
+BACKEND_SERVICES_OPEN_ENT = {"deployments", "useradm"}
```
Comment (lluiscampos) [inline extra/release_tool.py]: Unrelated to this PR, sorry, but aren't wee missing here `inventory-enterprise` update?

I think it should be something like:
```
BACKEND_SERVICES_OPEN = {"deviceauth"}
BACKEND_SERVICES_ENT = {"deployments-enterprise", "tenantadm", "useradm-enterprise", "inventory-enterprise"}
BACKEND_SERVICES_OPEN_ENT = {"deployments", "useradm", "inventory"}
```

Comment (tranchitella) [reply extra/release_tool.py]: Done.

File: testutils/infra/container_manager/docker_compose_manager.py (hunk 2 above)
Comment (tranchitella) [reply testutils/infra/container_manager/docker_compose_manager.py]: Ready to launch it, indeed.

File: component-maps.yml (hunk 1 above)
Comment (tranchitella) [reply component-maps.yml]: Done (without comment, I believe `release_componet: false` is self-explanatory).

**Comment by tranchitella** [comment]
@lluiscampos @kacf 
I force-pushed the changes you requested, and started a new pipeline:
https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/125256172

**Comment by lluiscampos** [comment]
:clap: :rocket:

**Pull request merged**